<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QWidget" name="MyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SNJ Code Deal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>13</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="tbtnDo">
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbtnClose">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>选择待烧录的fpga二进制文件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="editFile"/>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="tbtnSelect">
        <property name="text">
         <string>选择</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QProgressBar" name="pBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>13</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actSelect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/open1.png</normaloff>:/icons/icons/open1.png</iconset>
   </property>
   <property name="text">
    <string>选择目录</string>
   </property>
   <property name="toolTip">
    <string>选择目录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbtnClose</sender>
   <signal>clicked()</signal>
   <receiver>MyWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
